tests/regression/76-memleak/08-invalid-memcleanup-multi-threaded.c

```diff
@@ -7,6 +7,7 @@ int *m1;
 
 void *f1(void *arg) {
     m1 = malloc(sizeof(int));
+    free(m1);
     // Thread t1 leaks m1 here
     pthread_exit(NULL); //WARN
```

Comment:
The addition of this `free` changed the test from what it originally was.

Reply:
I removed the call to `free` now.